native/src/main/java/io/ballerina/stdlib/grpc/builder/BallerinaFileBuilder.java

```diff
@@ -65,16 +66,19 @@
     private static final String PROTO_FILE_EXTENSION = ".proto";
     public static Map<String, String> enumDefaultValueMap = new HashMap<>();
     public static Map<String, Boolean> dependentValueTypeMap = new HashMap<>();
+    public static Map<String, Class> streamClassMap;
 
     public BallerinaFileBuilder(byte[] rootDescriptor, Set<byte[]> dependentDescriptors) {
         setRootDescriptor(rootDescriptor);
         this.dependentDescriptors = dependentDescriptors;
+        streamClassMap = new HashMap<>();
     }
 
     public BallerinaFileBuilder(byte[] rootDescriptor, Set<byte[]> dependentDescriptors, String balOutPath) {
         setRootDescriptor(rootDescriptor);
         this.dependentDescriptors = dependentDescriptors;
         this.balOutPath = balOutPath;
+         streamClassMap = new HashMap<>();
```

Comment:
Seems not indented properly.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/560/commits/3a103d34283678d8df0c60cd70bcc311836ac637